environments/prod/helm/orchestration-workflows/encode/values.yaml

```diff
@@ -0,0 +1,15 @@
+enable: false
+argo:
+  env: prod
+  vaultPrefix: secret/dsde/monster/prod/command-center
+  namespace: encode
+  artifactBucket: broad-dsp-monster-encode-prod-argo-archive
+repo:
+  profileId: afc84cc8-e1ba-48d6-8153-dbc1d3f52c37
+  datasetId: 409b4c5e-564f-48fc-b590-cb4763234115
+  datasetName: broad_dsp_encode
+  dataProject: broad-datarepo-terra-prod-encd
+  url: https://jade-terra.datarepo-prod.broadinstitute.org
+chart:
+  git: false
+  ref: 1.0.0-M2
```

Comment:
Version explanation: milestone 2 of version 1 (milestone 1 was buggy). I'm using "milestone" here because I don't think we can consider the workflow a viable release candidate until it includes data-file ingest.